### ISSUE TITLE
Allow plotting sensor connectivity in 2D

### ISSUE THREAD

**Issue by hoechenberger** [mne-tools/mne-connectivity#97]
Currently, `plot_sensors_connectivity()` produces a 3D scene.

It would be great to have a visualization in 2D space too. With the new machinery in MNE-Python that powers [`plot_ch_adjacency()`](https://mne.tools/dev/generated/mne.viz.plot_ch_adjacency.html), this should be relatively easy to do / give some inspiration.

The plotting function would then return a Matplotlib `Figure` instead of a PyVista scene.

WDYT?

**Comment by ctrltz** [comment]
![image](https://user-images.githubusercontent.com/13971785/175323621-8dc819ab-5761-4cbc-a14b-1430c9c020aa.png)

Just a suggestion - it could also be cool to have so called head-in-head plots (fig. 5 of [this paper](https://www.frontiersin.org/articles/10.3389/fpsyg.2012.00166/full#h3), for example), seems possible with `mne.viz.iter_topography`

**Comment by hoechenberger** [comment]
I have to admit I don't understand the purpose of this head-in-head representation... what does it tell us?

**Comment by drammock** [comment]
I think each mini-head is connectivity based on a "seed" in that location. This is (somewhat redundantly) illustrated by the little black dot in each mini-head.

**Comment by hoechenberger** [comment]
So it shows connectivity "strength", e.g. in a simple case 1/number of edges between the starting node and the "target node"?

**Comment by ctrltz** [comment]
Yes, the idea is to display the whole NxN connectivity matrix at once as @drammock mentioned, black dot indeed might be not super-obvious when taken out of the context.

It has an advantage over graph-like display of important edges in case of many significant connections that might overlap and make the interpretation a little bit harder.

@hoechenberger you're right, all topomaps would be of the same color. And if, for example, connectivity between surroundings of C3 and C4 is very strong, then the topomap located at C3 would display high values around C4, and vice versa.
